packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.types.ts

```diff
@@ -1,16 +1,8 @@
 import { IColumn } from './DetailsList.types';
 import { IDetailsRowStyles, ICellStyleProps } from './DetailsRow.types';
-import { IBaseProps, IRefObject } from '../../Utilities';
 import { IDetailsListProps } from './DetailsList';
 
-export interface IDetailsRowFields {}
-
-export interface IDetailsRowFieldsProps extends Pick<IDetailsListProps, 'onRenderItemColumn'>, IBaseProps<IDetailsRowFields> {
-  /**
-   * Ref of component
-   */
-  componentRef?: IRefObject<IDetailsRowFields>;
```

Comment:
@JasonGore above said

> Practically speaking componentRef was pointing to an empty interface, so it was a useless prop anyways. Theoretically we could break somebody, but I don't think we'd be breaking any valid use cases as I don't see a practical use for componentRef.

I'm wondering, are there any cases in which we'd want to add to IDetailsRowFields in the future? What's the reasoning for removing this now? We do have lots of other places in the code that have empty `IComponent` interfaces. What was the point of adding those in the first place, just to make it extensible in the future?

Reply:
Just to be clear, we aren't removing `IDetailsRowField` anywhere, we're just removing `componentRef` prop from it.

I noticed it was in the create-component script before I modified it by default. I suspect this is why there are empty interfaces all over.

David and I talked about it and agree that `componentRef` shouldn't be added by default. It should be opt-in and the way it's used is now different per #8169.

Reply:
@JasonGore are we OK with the removal in this PR as it is "dead-code"?